code/modules/mob/living/living.dm

```diff
@@ -432,9 +432,10 @@
 
 /mob/living/verb/succumb(whispered as null)
 	set hidden = TRUE
-	if (!CAN_SUCCUMB(src))
-		to_chat(src, text="You are unable to succumb to death! This life continues.", type=MESSAGE_TYPE_INFO)
+	if (whispered && !CAN_WHISPER_SUCCUMB(src))
+		to_chat(src, text="You cannot succumb while whispering!", type=MESSAGE_TYPE_INFO)
```

Comment:
This message doesn't seem clear to me. I think you should keep the old one but display a special message if they have the rarer trait.